File: src/test/java/com/flexpoker/test/util/datageneration/DeckGenerator.java
package com.flexpoker.test.util.datageneration;

import java.util.Arrays;

import com.flexpoker.table.command.CardsUsedInHand;
import com.flexpoker.table.command.PocketCards;

public class DeckGenerator {

    public static CardsUsedInHand createDeck() {
        return new CardsUsedInHand(CardGenerator.createFlopCards(),
                CardGenerator.createTurnCard(), CardGenerator.createRiverCard(),
                Arrays.asList(new PocketCards[] { CardGenerator.createPocketCards1(),
                        CardGenerator.createPocketCards2() }));
    }
}

File: src/test/kotlin/com/flexpoker/test/util/datageneration/DeckGenerator.kt
package com.flexpoker.test.util.datageneration

import com.flexpoker.table.command.CardsUsedInHand

object DeckGenerator {

    fun createDeck(): CardsUsedInHand {
        return CardsUsedInHand(
            CardGenerator.createFlopCards(),
            CardGenerator.createTurnCard(),
            CardGenerator.createRiverCard(),
            listOf(CardGenerator.createPocketCards1(), CardGenerator.createPocketCards2())
        )
    }

}